### PR TITLE
Outline for Journal: Week 4

### PR DESCRIPTION
**Pull request by cmauban**

### CODE REVIEW

**Comment by ericksonk** [comment]
Detailed and helpful tutorial thus far :+1: 
Curse the chessboard!!!

**Comment by s4dmw** [comment]
journal - i'm coming to share the same feelings about scss vs css. it seemed super useful at first but i think i just prefer straight css at this point.

tutorial - good topic! i've been using the scripts "short cuts" a ton over the last week...  :+1:

**Comment by jeremyrist** [comment]
:+1: I've yet to approach this browser-sync monster....command+R for me right now haha

File: tutorial-week-4.md [hunk 1]
```diff
@@ -1,8 +1,29 @@
-##Tutorial: Package.json
+##Tutorial: Adding a Package.json file
 
-- `cd` folder, correct branch
-- `npm init` (gives you package.json)
-- install development dependencies
-- in `scripts` key add: "start"- browser-sync, "sass"- node-sass
+###Step 1:
```
Comment (al-the-x) [inline tutorial-week-4.md:3]: Before you get into the meat of the tutorial, be sure to explain what the purpose of a `package.json` is: what problem is this a solution to?

File: tutorial-week-4.md [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+in the terminal..
+```
+npm install --save-dev mocha chai
+npm install --save-dev browser-sync
+npm install --save-dev node-sass
+```
+
+###Step 4: Add `scripts` keys
+Under `scripts` key add: "start"- browser-sync, "sass"- node-sass
+..gonna add the right values
```
Comment (al-the-x) [inline tutorial-week-4.md:25]: Don't forget to explain _why_ we need those scripts...!

**Pull request merged**